views/index.jade

```diff
@@ -86,5 +89,16 @@ block content
             a(href='http://yaydoc.fossasia.org' target='_blank') yaydoc Documentation
         .modal-footer
           a.btn.btn-default(id='onetimeDeploy') Proceed with One-time Deployment
+  #detailed-logs-modal.modal.fade(role='dialog')
+    .modal-dialog
+      .modal-content
+        .modal-header
+          button.close(type='button', data-dismiss='modal') ×
+          h5.modal-title Detailed Logs
+        .modal-body
+          pre#detailed-logs.pre-scrollable
+        .modal-footer
+          button.btn.btn-default#copy-button(data-clipboard-action='copy' data-clipboard-target='#detailed-logs') Copy to Clipboard
+          a#downloadDetailedLogs.btn.btn-default Download
```

Comment:
Keep the `copy to clipboard` button on the far left instead of stacking both buttons together